delivery_service.rb

```diff
@@ -1,2 +1,37 @@
+require_relative 'car'
+require_relative 'bike'
+require_relative 'vehicle_availabilities'
+
+
 class DeliveryService
-end
\ No newline at end of file
+  extend VehicleAvailabilities
+
+  def initialize
+    @cars = DeliveryService.availability
```

Comment:
we don`t need hard coded avability count
also variable name @cars is not correct cause DeliveryService.availability provides bikes and cars

Reply:
removed